packages/react-core/src/demos/CardDemos.md

```diff
@@ -260,3 +253,163 @@ class CardGridDemo extends React.Component {
   }
 }
 ```
+
+### Horizontal split
+
+```js
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  CardFooter,
+  Grid,
+  GridItem,
+  Button
+} from '@patternfly/react-core';
+
+CardHorizontalSplitDemo = () => {
+  return (
+    <Card id="card-demo-horizontal-split-example" isFlat>
+      <Grid md={6} hasGutter>
+        <GridItem style={{minHeight: '200px', backgroundPosition: 'center', backgroundSize: 'cover', backgroundImage: 'url(/assets/images/pfbg_992@2x.jpg)' }}/>
+        <GridItem>
+          <CardTitle>Headline</CardTitle>
```

Comment:
The `<Card*>` elements should be wrapped in `<Card>`. We're missing this on the core side, too - https://github.com/patternfly/patternfly/issues/4310

Reply:
@mcoker I am confused about the structure.  I assumed the `<Card>` on line 273 was the wrapping  element here.

Reply:
Ah hah! I missed that, ignore my comment 🤦‍♂️